applications/search/src/pages/search-page/results-api/results-api.component.jsx

```diff
@@ -202,28 +202,26 @@ export class ResultsApi extends React.Component {
           <div id="apis" className="col-12 col-lg-8">
             {renderHits(apiItems, publishers)}
 
-            {_.get(apiItems, 'total', 0) > 10 && (
-              <div className="col-12 d-flex justify-content-center">
-                <span className="uu-invisible" aria-hidden="false">
-                  Sidepaginering.
-                </span>
-                <ReactPaginate
-                  pageCount={pageCount}
-                  pageRangeDisplayed={2}
-                  marginPagesDisplayed={1}
-                  previousLabel={localization.page.prev}
-                  nextLabel={localization.page.next}
-                  breakLabel={<span>...</span>}
-                  breakClassName="break-me"
-                  containerClassName="pagination"
-                  onPageChange={onPageChange}
-                  subContainerClassName="pages pagination"
-                  activeClassName="active"
-                  initialPage={page}
-                  disableInitialCallback
-                />
-              </div>
-            )}
+            <div className="col-12 d-flex justify-content-center">
```

Comment:
Skulle ikke denne vært her på siden for alle fanene, men at du sjekker at det finnes flere enn antall som er valgt i max visning? Eks. {_.get(apiItems, 'total', 0) > [verdien som er valgt i nedtrekk for treff pr. side] && (

Reply:
Her må [1] vises hvis det er bare én side. Siden Previous og Next er disabled, så vises ikke de. <ReactPaginate må altså alltid vises